app/views/candidate_interface/decoupled_references/review/confirm_delete_request.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :title, t('page_titles.references_delete_request') %>
+<% content_for :before_content, govuk_back_link_to %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with model: @reference, url: candidate_interface_destroy_decoupled_reference_path(@reference), method: :delete do |f| %>
+      <h1 class="govuk-heading-xl">
+        <%= t('page_titles.references_delete_request') %>
+      </h1>
+
+      <%= f.submit 'Yes I’m sure - delete this reference request' , class: 'govuk-button govuk-button--warning', data: { module: 'govuk-button' } %>
```

Comment:
Should button labels be in the localisation file?

Reply:
Have done 👍